README.md

```diff
@@ -98,7 +98,8 @@ There’s also a <a href="https://vimeo.com/168648012">fantastic video</a> on ho
 ## Quick start
 
 1. Clone this repo using `git clone --depth=1 https://github.com/mxstbr/react-boilerplate.git`
-1. Run `npm run setup` to install dependencies and clean the git repo.<br />
+1. As an alternative to npm, you can use yarn for dependency management, which might make the installation process easier for different platforms to do so: `USE_YARN=true npm run setup`. If you don't have `yarn` installed check out <a href="https://yarnpkg.com/en/docs/install" target="_blank">here</a> how to install yarn on your OS or do a `npm i -g yarn`
+1. (Skip this if you have done setup with `yarn`). Run `npm run setup` to install dependencies and clean the git repo.<br />
```

Comment:
Instead of talking about this long and wide, what about detecting if people have yarn installed in the setup script and then just using it if they do? Why should they need to manually do `USE_YARN=true`? (which, btw, only works on unix based systems, on Windows it's different)